debian/control

```diff
@@ -2,7 +2,7 @@ Source: obs-service-tar-scm
 Section: devel
 Priority: optional
 Maintainer: Daniel Gollub <dgollub@brocade.com>
-Build-Depends: debhelper (>= 7.0.0), python3, python3-dateutil, dh-python
+Build-Depends: debhelper (>= 7.0.0), python3, python-argparse, python3-dateutil, dh-python
```

Comment:
is this still required? argparse is part of the python core modules since python 3.2 and we will not support python2 any longer.

Reply:
Until Debian Bookworm ships without python2 I believe this still needed. Note this only for the debian package and does not impact the rpm for suse.

Reply:
With Bookworm now available I've dropped this dependency.